ci/pipeline.yml

```diff
@@ -20,6 +20,7 @@ steps:
   - label: '{{emoji}}{{engine}} {{suite}}/{{workload}}/{{platform}} BATCH_SIZE={{batch_size}} '
     command: |
       {{python}} -m pip install pyyaml --user
+      {{python}} -m pip install numpy --user
```

Comment:
You can put multiple deps on a single line. But when I tried to do this I got horrible errors, perhaps because I needed to pin numpy to an earlier version.

```
{{python}} -m pip install numpy pyyaml --user
```